qemu/tests/live_backup_base.py

```diff
@@ -145,9 +149,20 @@ def get_image_size(self, image):
         :param image: image file.
         :return: image size.
         """
+        force_share = False
+        pause_vm = False
         params = self.params.object_params(image)
         qemu_image = qemu_storage.QemuImg(params, self.data_dir, image)
-        image_info = qemu_image.info()
+        if self.vm:
+            pids = process.getoutput(
+                "lsof %s |grep -v PID|awk '{print $2}'" % qemu_image.image_filename)
```

Comment:
better to simply use `"lsof -t %s"`

Reply:
@lolyu  thanks your comments,  "-t" options  is not always supported.